cityofaustin/atd-moped
# Create shared use trail -  dual path component

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14518

## Testing
**URL to test:** Local

**Steps to test:**
1. Create a new map component using: `Trail - Shared use (Paved Dual Path)`
2. Check out the work type and subcomponent options. they should match the component called `Trail - Shared Use Path (Paved)`

---
#### Ship list
